share/spice/just_delete_me/footer.handlebars

```diff
@@ -0,0 +1,10 @@
+{{#if delete_url}}
+    <div class="one-line">
+         <a href='{{delete_url}}'>{{delete_url}}</a>
```

Comment:
Let's show descriptive text instead of these URLs.

I think the footer should look like this:

What are your thoughts?

![how_to_delete_cloud_account_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/14183988/f1db34de-f740-11e5-9aa3-84dba147188b.png)